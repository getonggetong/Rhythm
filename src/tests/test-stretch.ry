init() {
	
	print(A4 -> 3);
	print([A4,B4] -> 3);
	print([[A4,B4]] -> 3);
}