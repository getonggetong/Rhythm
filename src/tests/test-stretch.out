[A4,A4,A4]
[A4,B4,A4,B4,A4,B4]
[[A4,B4],[A4,B4],[A4,B4]]